activesupport/lib/active_support/dependencies.rb

```diff
@@ -736,6 +736,7 @@ def remove_constant(const) #:nodoc:
         begin
           constantized = parent.const_get(to_remove, false)
         rescue NameError
+          # Skip when const is unreachable.
```

Comment:
I wold like to emphasize to the reader that this is something that may happen adding a "just". Not a native speaker but I think it conveys that nuance. Something like "The constant is no longer reachable, just skip it."

Also "no longer" reminds the reader (who may not have all the context in their head), that this was reachable at some point, which is just a bit more informative.

Also note the alternative uses "constant" instead of "const", comments use regular English.